packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -135,7 +135,10 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
     () => mergeButtonBaseStyles(props.increaseFlyoutItemSize ? devicesButtonWithIncreasedTouchTargets : {}),
     [props.increaseFlyoutItemSize]
   );
-
+  // when props.options is false then we want to hide the whole control bar.
+  if (props.options === false) {
+    return <></>;
+  }
```

Comment:
I'd recommend moving line 97 below this  and changing it to

```ts
const options = props.options === true ? {} : props.options
```

to better reflect what's happening.